lib/cricket-cup/addon/pods/components/cricket-match-card/template.hbs

```diff
@@ -65,7 +65,6 @@
     {{#if match.resultDeclared}} 
     <ResultView 
       @predictions={{predictions}}
```

Comment:
won't work

Reply:
we are not using earnings from the parent anymore! we'll be computing the earnings using the predictions